components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/RecoveryApiV2.java

```diff
@@ -96,10 +102,58 @@ public List<AccountRecoveryType> initiatePasswordRecovery(RecoveryInitRequest re
     public RecoveryResponse recoverPassword(RecoveryRequest recoveryRequest, String tenantDomain,
                                             Map<String, String> headers) throws ApiException {
 
-        String localVarPath = "/password/recover".replaceAll("\\{format\\}", "json");
+        String localVarPath = "/password/recover";
         return recover(recoveryRequest, tenantDomain, headers, localVarPath);
     }
 
+    /**
+     * Resend password recovery confirmation details.
+     *
+     * @param resendRequest   Resend request. (required)
+     * @param tenantDomain    Tenant Domain which user belongs. Default "carbon.super". (optional)
+     * @param headers         Any additional headers to be embedded. (optional)
+     * @return Resend response.
+     * @throws ApiException If fails to make API call.
+     */
+    public ResendResponse resendPasswordNotification(ResendRequest resendRequest, String tenantDomain,
+                                                     Map<String, String> headers) throws ApiException {
+
+        String localVarPath = "/password/resend";
```

Comment:
shall we get these to constants?. Check other places as well

Reply:
addressed with https://github.com/wso2/carbon-identity-framework/pull/5601/commits/b1c3d31e9ef460ab19ff45869ebe4e8dec8454d8